packages/paste-core/components/anchor/src/index.tsx

```diff
@@ -1,48 +1,53 @@
 import * as React from 'react';
+import * as PropTypes from 'prop-types';
 import {StyledLink} from './styles';
-import {AnchorProps} from './types';
+
+export type AnchorTargets = '_self' | '_blank' | '_parent' | '_top';
+export type AnchorTabIndexes = 0 | -1;
+
+interface Anchor {
+  className?: never;
+  children: NonNullable<React.ReactNode>;
+  href: string;
+  id?: never;
+  onBlur?(event: React.FocusEvent<HTMLElement>): void;
+  onClick?(event: React.MouseEvent<HTMLElement>): void;
+  onFocus?(event: React.FocusEvent<HTMLElement>): void;
+  rel?: string;
+  tabIndex?: AnchorTabIndexes;
+  target?: AnchorTargets;
+}
 
 const EXTERNAL_LINK_REGEX = /^(https?:)[^\s]*$/;
```

Comment:
Since we're not using the word "Link" for this component, maybe we can rename this to `EXTERNAL_URL_REGEX`